### PR TITLE
chore: feat: remove theia from dashboard; change dashboard notice to removal (not deprecation) (CRW-3489, CRW-3663)

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: feat: remove theia from dashboard; change dashboard notice to removal (not deprecation) (CRW-3489, CRW-3663) #408

Change-Id: I98316ebe60801f31172e852655eccfba49fe55da
Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-3489 
https://issues.redhat.com/browse/CRW-3663 

See also 

https://issues.redhat.com/browse/CRW-4077

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by nickboldt** [comment]
See also https://github.com/redhat-developer/devspaces/pull/911

**Review by amisevsk — APPROVED** [review]

**Review by amisevsk — APPROVED** [review]

**Review by svor — APPROVED** [review]
LGTM

File: devspaces-operator-bundle/build/scripts/sync-che-olm.sh [hunk 1]
```diff
@@ -323,12 +323,10 @@ for CSVFILE in ${TARGETDIR}/manifests/devspaces.csv.yaml; do
...
 		# TODO also remove theia from docs section #selecting-a-workspace-ide & related tables
-	# CRW-3488 (3.4 issue), CRW-3405 (3.4 RN) add Theia deprecation warning to dashboard
-	headerMessage="Microsoft Visual Studio Code - Open Source is the default <a href='https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/${DS_VERSION}/html-single/user_guide/index#selecting-a-workspace-ide'>editor</a> for new workspaces. Eclipse Theia is <a href='https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/${DS_VERSION}/html-single/release_notes_and_known_issues/index#deprecated-functionality-crw-3663'>deprecated</a> and will be removed in a future release."
+	headerMessage="Microsoft Visual Studio Code - Open Source is the default <a href='https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/${DS_VERSION}/html-single/user_guide/index#selecting-a-workspace-ide'>editor</a> for new workspaces. Eclipse Theia is <a href='https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/${DS_VERSION}/html-single/release_notes_and_known_issues/index#removed-functionality-crw-3489'>no longer supported</a>."
```
Comment (nickboldt) [inline devspaces-operator-bundle/build/scripts/sync-che-olm.sh:329]:
```suggestion
	headerMessage="Microsoft Visual Studio Code - Open Source is the default <a href='https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/${DS_VERSION}/html-single/user_guide/index#selecting-a-workspace-ide'>editor</a> for new workspaces. Eclipse Theia is <a href='https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/${DS_VERSION}/html-single/release_notes_and_known_issues/index#removed-functionalities'>no longer supported</a>."
```

File: devspaces-operator-bundle/build/scripts/sync-che-olm.sh [hunk 2]
```diff
@@ -323,12 +332,10 @@ for CSVFILE in ${TARGETDIR}/manifests/devspaces.csv.yaml; do
...
```
Comment (nickboldt) [inline devspaces-operator-bundle/build/scripts/sync-che-olm.sh:338]:
```suggestion
	headerMessage="Microsoft Visual Studio Code - Open Source is the default <a href='https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/${DS_VERSION}/html-single/user_guide/index#selecting-a-workspace-ide'>editor</a> for new workspaces. Eclipse Theia is <a href='https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/${DS_VERSION}/html-single/release_notes_and_known_issues/index#removed-functionalities'>no longer supported</a>."
```

**Pull request merged**